.github/workflows/pre-commit-hooks.yml

```diff
@@ -12,6 +12,6 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: '3.x'
```

Comment:
> 3.x means version is python3? 

Yes, specifically the latest stable version.

> do not need to use spefic a pinted version here?

Not anymore. We pinned it to 3.10 because pylint was not fully compatible with 3.11 (something about the `os` module not having a `environ` attribute). The latest release of pylint no longer shows this problem, so I went back to using the latest python3 version. I also did `pre-commit autoupdate`.